src/options/options.html

```diff
@@ -79,17 +90,46 @@
             </p>
           </div>
         </p>
+        </div>
       </p>
       <p>
-        <div class="sectionTitle">Test your script</div>
-        <p>
-          Test your script by sending "true" to simulate that there are unread messages or "false"
-          to simulate there are no more. Also make sure your script reacts properly if more than
-          one "true" or more than one "false" are sent in a row.
-        </p>
+        <div class="sectionTitle">Select type of data sent to the native script</div>
         <p>
-          <button type="button" class="smallBtn" id="notifyScriptTrue">true</button>
-          <button type="button" class="smallBtn" id="notifyScriptFalse">false</button>
+          <div>
+            <p>
+              <input type="radio" id="notifyScriptSimple" name="notifyScriptType" value="simple" checked="checked">
+              <label for="notifyScriptSimple">
+                <strong>Simple</strong>
+                <br/>
+                If there are unread messages, the script receives a "true" whenever a new message
+                is received, read or deleted, otherwise a "false", if there are no new messages.
+                The script must react properly if more than one "true" or "false" are sent in a
+                row.
+              </label>
+            </p>
+            <p>
+              Test:
+              <button type="button" class="smallBtn" id="notifyScriptTrue">true</button>
+              <button type="button" class="smallBtn" id="notifyScriptFalse">false</button>
+            </p>
+            <p>
+              <input type="radio" id="notifyScriptExtended" name="notifyScriptType" value="extended">
+              <label for="notifyScriptExtended">
+                <strong>Extended</strong>
+                <br/>
+                For each new, read or deleted message, the script receives some information about
+                the message (sender, receiver, subject, date received, mailbox and the new status)
+                beside the overall number and the number of unread messages for each watched inbox.
+              </label>
+            </p>
+            <p>
+              Test:
+              <button type="button" class="smallBtn" id="notifyScriptStart">Start Thunderbird</button>
+              <button type="button" class="smallBtn" id="notifyScriptNew">New message</button>
+              <button type="button" class="smallBtn" id="notifyScriptRead">Message read</button>
+              <button type="button" class="smallBtn" id="notifyScriptDeleted">Message deleted</button>
```

Comment:
Is this "deleted" test still required?

Reply:
Aargh! You are absolutely right.

Done.